benchalerts/tests/integration_tests/test_pipeline_steps/test_conbench.py

```diff
@@ -15,54 +15,71 @@
 import pytest
 from benchclients.conbench import LegacyConbenchClient
 
-from benchalerts.pipeline_steps.conbench import GetConbenchZComparisonStep
+from benchalerts.pipeline_steps.conbench import (
+    BaselineRunCandidates,
+    GetConbenchZComparisonStep,
+)
 
 ConbenchClient = LegacyConbenchClient
 
 
 @pytest.mark.parametrize(
-    ["conbench_url", "commit", "expected_len", "expected_bip"],
+    ["conbench_url", "commit", "baseline_type", "expected_len", "expected_bip"],
     [
         # baseline is parent
         (
             "https://conbench.ursa.dev/",
             "bc7de406564fa7b2bcb9bf055cbaba31ca0ca124",
+            BaselineRunCandidates.parent,
             8,
             True,
         ),
         # baseline is not parent
         (
             "https://velox-conbench.voltrondata.run",
             "2319922d288c519baa3bffe59c0bedbcb6c827cd",
+            BaselineRunCandidates.fork_point,
             1,
             False,
         ),
         # no baseline
         (
             "https://velox-conbench.voltrondata.run",
             "b74e7045fade737e39b0f9867bc8b8b23fe00b78",
+            BaselineRunCandidates.latest_default,
             1,
             None,
         ),
         # errors
         (
             "https://conbench.ursa.dev",
             "9fa34df27eb1445ac11b0ab0298d421b04be80f7",
+            BaselineRunCandidates.parent,
             7,
             True,
         ),
     ],
 )
 def test_GetConbenchZComparisonStep(
-    monkeypatch: pytest.MonkeyPatch, conbench_url, commit, expected_len, expected_bip
+    monkeypatch: pytest.MonkeyPatch,
+    conbench_url,
+    commit,
```

Comment:
Let's rename this to `commit_hash`, for consistency and to distinguish this from more complex commit objects.

Reply:
Also, maybe a type annotation? I was literally just wondering what kind of object `commit` might be, and of course this might be addressed via type annotation and/or more explicit naming.

Reply:
oh good calls, done